README.md

```diff
@@ -1,13 +1,45 @@
-# The Uno Platform ToDo reference App
+<p align="center">
+  <img src="doc/assets/To_Do_App_Banner.png">
```

Comment:
@agneszitte-nventive  I would call it Uno Platform "Uno To Do" reference app 

we are writing it as not concatenated.